test/runtests.jl

```diff
@@ -19,8 +21,16 @@ using Test
             @test resp.code == 226
         end
     end
-    @testset "ssl - $mode" for mode in (:explicit, :implicit)
-        FTPServer.serve(; security=mode) do server
+    @testset "no-ssl debug true" begin
```

Comment:
Should we also include the upload test we used to reproduce this problem?

Reply:
I wasn't sure if we wanted to add that here or in a separate PR

Reply:
Ah ok, lets just not forget